src/main/java/com/onlinebookstore/onlinebookstore/model/Book.java

```diff
@@ -0,0 +1,40 @@
+package com.onlinebookstore.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+public class Book implements Cloneable {

```

Comment:
What is the reason to make of clonable?

Reply:
It a leftovers from first raw version, I will delete it